Tests/BotTests/GitHub/GitHubAPITests.swift

```diff
@@ -2,6 +2,18 @@ import XCTest
 import Nimble
 @testable import Bot
 
+extension Result {
+
+    var value: Success? {
```

Comment:
[nit] This can also be achieved directly with the built-in `try? result.get()` instead of `result.value`, so just wondering if that extension is really necessary? 🤔 (but hey if it's more readable at call site and only extended for tests anyway, it's ok to keep it)